docs/resources/aws_ssm_associations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: About the aws_ssm_associations Resource
+platform: aws
+---
+
+# aws\_ssm\_associations
+
+Use the `aws_ssm_associations` InSpec audit resource to test properties of a collection of AWS SSM Associations.
+
+## Syntax
+
+ Ensure you have exactly 3 associations
+
+    describe aws_ssm_associations do
+      its('names.count') { should cmp 3 }
+    end
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on SSM](https://docs.aws.amazon.com/systems-manager/?id=docs_gateway).
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|association\_ids             | Provides the ID of the association. |
+|association\_names           | Provides the name of the association. |
```

Comment:
This should be `names`.

Reply:
An 'association_name' is the name of the association, the 'name' field is the document name that is linked to the association. I think it would be useful to keep both.